hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/datanode/UsageInfoSubcommand.java

```diff
@@ -96,19 +97,37 @@ public void execute(ScmClient scmClient) throws IOException {
    * @param info Information such as Capacity, SCMUsed etc.
    */
   public void printInfo(HddsProtos.DatanodeUsageInfoProto info) {
-    double capacity = (double) info.getCapacity();
-    double usedRatio = info.getUsed() / capacity;
-    double remainingRatio = info.getRemaining() / capacity;
+    long capacity = info.getCapacity();
+    long used = info.getUsed(), remaining = info.getRemaining();
+    long totalUsed = capacity - remaining;
+    double usedRatio = used / (double) capacity;
+    double remainingRatio = remaining / (double) capacity;
     NumberFormat percentFormat = NumberFormat.getPercentInstance();
-    percentFormat.setMinimumFractionDigits(5);
+    percentFormat.setMinimumFractionDigits(2);
+    percentFormat.setMaximumFractionDigits(2);
 
     System.out.printf("Usage info for datanode with UUID %s:%n",
         info.getNode().getUuid());
-    System.out.printf("%-10s: %20sB %n", "Capacity", info.getCapacity());
-    System.out.printf("%-10s: %20sB (%s) %n", "SCMUsed", info.getUsed(),
+    // print capacity in a readable format
+    System.out.printf("%-13s: %-21s (%s) %n", "Capacity", capacity + " B",
```

Comment:
Let's not print capacity and other data in bytes. GB is sufficient.

Reply:
Thanks for reviewing. Had a discussion with @mukul1987 regarding this in which we had decided to use both the units(like `hdfs dfsadmin -report`).

Reply:
I see. Can we remove the space? I think removing -21 would make it more readable.

Reply:
Sure. I'm only doing that to keep the information inside the brackets in one single column.